### PR TITLE
[1LP][RFR] Login sentaku

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit adaabb9086fcd8a9f272b42e4c5ea783bc9b1138:

`cfme/tests/login/test_ssui_login.py`:
- :red_circle: Line 4:1: **F401** *'cfme.utils.appliance.get_or_create_current_appliance' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/login/test_ssui_login.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+@pytest.mark.uncollectif(lambda: current_version() < '5.8')
+@pytest.mark.parametrize('context', [ViaSSUI])
+def test_ssui_login(appliance, context):
+    """SSUI Login test"""
+
+    with appliance.context.use(context):
+        appliance.server.login()
+        logged_in_page = appliance.browser.create_view(SSUIBaseLoggedInPage)
```
Comment (RonnyPfannschmidt) [inline cfme/tests/login/test_ssui_login.py]: at first glance i wonder about using `appliance.browser` here i was under the impression ssui should use a different one in some way

@psav ?

Comment (psav) [reply cfme/tests/login/test_ssui_login.py]: Agreed, I believe it's called ```appliance.ssui``` but in all honesty, I forget what the ```context``` is......but we should be instantiating the view from the context, IF the test is to be parametrized on the context. This test, as it is just testing ssui login, doesn't need to use context as a parameter. If only makes sense if we are trying to do both UI and SSUI.

**Comment by dajoRH** [comment]
Lint report for commit a9218563ff4b1017b92debb322898316c67daa85:

`cfme/tests/test_login.py`:
- :red_circle: Line 53:13: **F841** *local variable 'logged_in_page' is assigned to but never used*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Pull request merged**